Makefile

```diff
@@ -281,199 +281,101 @@ verify-extended: check-generate check format test-cov test-cov-clean test-integr
-	@# cleanup namespaces that don't get deleted automatically
-	$(KUBECTL) delete ns gardener-system-seed-lease istio-ingress istio-system --ignore-not-found
-
-gardener-ha-single-zone-up: $(SKAFFOLD) $(HELM) $(KUBECTL)
-	SKAFFOLD_DEFAULT_REPO=localhost:5001 SKAFFOLD_PUSH=true $(SKAFFOLD) run -p ha-single-zone
-
-gardener-ha-multi-zone-up: $(SKAFFOLD) $(HELM) $(KUBECTL)
-	SKAFFOLD_DEFAULT_REPO=localhost:5001 SKAFFOLD_PUSH=true $(SKAFFOLD) run -p ha-multi-zone
-
-gardener-ha-single-zone-down: $(SKAFFOLD) $(HELM) $(KUBECTL)
-	@# delete stuff gradually in the right order, otherwise several dependencies will prevent the cleanup from succeeding
-	$(KUBECTL) delete seed local-ha-single-zone --ignore-not-found --wait --timeout 5m
-	$(SKAFFOLD) delete -m provider-local,gardenlet -p ha-single-zone
-	$(KUBECTL) delete validatingwebhookconfiguration/gardener-admission-controller --ignore-not-found
-	$(KUBECTL) annotate project local garden confirmation.gardener.cloud/deletion=true
-	$(SKAFFOLD) delete -m local-env -p ha-single-zone
-	$(SKAFFOLD) delete -m etcd,controlplane -p ha-single-zone
-	@# workaround for https://github.com/gardener/gardener/issues/5164
-	$(KUBECTL) delete ns seed-local-ha-single-zone --ignore-not-found
-	@# cleanup namespaces that don't get deleted automatically
-	$(KUBECTL) delete ns gardener-system-seed-lease istio-ingress istio-system --ignore-not-found
-
-gardener-ha-multi-zone-down: $(SKAFFOLD) $(HELM) $(KUBECTL)
-	@# delete stuff gradually in the right order, otherwise several dependencies will prevent the cleanup from succeeding
-	$(KUBECTL) delete seed local-ha-multi-zone --ignore-not-found --wait --timeout 5m
-	$(SKAFFOLD) delete -m provider-local,gardenlet -p ha-multi-zone
-	$(KUBECTL) delete validatingwebhookconfiguration/gardener-admission-controller --ignore-not-found
-	$(KUBECTL) annotate project local garden confirmation.gardener.cloud/deletion=true
-	$(SKAFFOLD) delete -m local-env -p ha-multi-zone
-	$(SKAFFOLD) delete -m etcd,controlplane -p ha-multi-zone
-	@# workaround for https://github.com/gardener/gardener/issues/5164
-	$(KUBECTL) delete ns seed-local-ha-multi-zone --ignore-not-found
-	@# cleanup namespaces that don't get deleted automatically
-	$(KUBECTL) delete ns gardener-system-seed-lease istio-ingress istio-system --ignore-not-found
-
-gardenlet-kind2-up: $(SKAFFOLD) $(HELM)
-	$(SKAFFOLD) deploy -m kind2-env -p kind2 --kubeconfig=$(GARDENER_LOCAL_KUBECONFIG)
-	@# define GARDENER_LOCAL_KUBECONFIG so that it can be used by skaffold when checking whether the seed managed by this gardenlet is ready
-	GARDENER_LOCAL_KUBECONFIG=$(GARDENER_LOCAL_KUBECONFIG) SKAFFOLD_DEFAULT_REPO=localhost:5001 SKAFFOLD_PUSH=true $(SKAFFOLD) run -m provider-local,gardenlet -p kind2
-
-gardenlet-kind2-down: $(SKAFFOLD) $(HELM)
-	$(SKAFFOLD) delete -m kind2-env -p kind2 --kubeconfig=$(GARDENER_LOCAL_KUBECONFIG)
-	$(SKAFFOLD) delete -m gardenlet,kind2-env -p kind2
-
+	./hack/gardener-down.sh
 register-local-env: $(KUBECTL)
```

Comment:
```suggestion

register-local-env: $(KUBECTL)
```

Reply:
I have intentionally grouped all this together for easier maintenance so that related make targets/rules are not scattered around the file :)